.travis.yml

```diff
@@ -0,0 +1,25 @@
+dist: bionic
+sudo: required
+arch: arm64
+
+language: go
+
+os:
+- linux
+
+go:
+  - "1.13.7"
+
+script:
+  - uname -a
+  - sudo apt-get install build-essential openjdk-8-jdk python zip unzip -y
+  - pwd
+  - mkdir bazel
+  - cd bazel
+  - wget https://github.com/bazelbuild/bazel/releases/download/2.1.0/bazel-2.1.0-dist.zip
+  - unzip bazel-2.1.0-dist.zip
+  - bash ./compile.sh
```

Comment:
I don't like the idea to compile bazel here. I think we can use a fedora docker container and skip this step:
https://github.com/avagin/gvisor/commit/b01da6bbb52674dbfe59a4922b5d80146ea40910

Reply:
I see.
It looks like this PR is duplicated with yours.
Le me close my PR.